content/installation/prerequisites.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: bt_wiki
+title: Prerequisites for Installing a Cloudify Manager
+category: Installation
+draft: false
+weight: 300
+---
+
+Before you install a Cloudify Manager, you must review these prerequisites and make sure that your environment is prepared to support the Cloudify Manager.
+
+## Architecture and OS
+
+Cloudify Manager must run on a 64-bit host with RHEL/CentOS 7.x or higher.
+
+## Manager Resources
+
+Cloudify Manager requires at the least:
+
+ -       | Minimum | Recommended |
+---------|---------|-------------|
+ vCPUs   | 2       | 16          |
+ RAM     | 4GB     | 8GB         |
```

Comment:
at some point we need to review this, because this has been much lower ever since we got rid of elasticsearch

Reply:
Eh. I guess we could go lower than the minimum, but I think 2 vCPUs and 4GB RAM is a more than a reasonable assumption for a production environment. We maybe can add, like, a "testing only" section with lower reqs.

Reply:
@simon-bar Do you want to lower the minimum requirements or add "testing" requirements?

Reply:
@sivanbar Do you want to lower the minimum requirements or add "testing" requirements?